Formula/vcpkg.rb

```diff
@@ -1,9 +1,9 @@
 class Vcpkg < Formula
   desc "C++ Library Manager"
   homepage "https://github.com/microsoft/vcpkg"
-  url "https://github.com/microsoft/vcpkg-tool/archive/2022-07-14.tar.gz"
-  version "2022.07.14"
-  sha256 "e7f4783d0c30c074029a08cf83443838dd3cee1610458ce2fc0fa9a8f7f4411d"
+  url "https://github.com/microsoft/vcpkg-tool/archive/2022-07-21.tar.gz"
+  version "2022-07-21"
```

Comment:
Any reason we're changing from `.`s to `-`s?

Reply:
Just matching upstream and livecheck

Reply:
Last I checked their actual `vcpkg` build, I think `--version` used `.`s though. Did they change that (or am I misremembering)?

Reply:
```
  ~ vcpkg --version
vcpkg package management program version 2022-07-14-2022.07.14
```

Reply:
That's not confusing at all 😓

Reply:
Other package managers kept the dots: https://repology.org/project/vcpkg/versions.
I would rather keep them too.